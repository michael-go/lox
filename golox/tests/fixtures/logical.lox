print "hi" or 2;
print nil or "yes";

if ("foo" == "fo" + "o") {
    print "bar";
} else {
    print "baz";
}