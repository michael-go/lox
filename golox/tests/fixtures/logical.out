# exit code: 0
# stdout:
hi
yes
bar

# stderr:

